src/styles/index.scss

```diff
@@ -1,3 +1,152 @@
-body {
+.body {
   margin: 0;
 }
+
+.button {
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  margin: 16px;
+  background: #00acdc;
+  border: 0;
+  align-items: center;
```

Comment:
The 'align-items' property should not be used for '.button' as it is not a flex or grid container. Consider removing it if it's not necessary.